projects/assets/src/translations/en/common.ts

```diff
@@ -21,6 +21,9 @@ export const common = {
     search: 'Search',
     add: 'Add',
   },
+  aria: {
```

Comment:
Just wondering why this is here? I'm not seeing it used anywhere.

Reply:
So I was using it initially, but then I removed the aria attribute that used it since it was not working as intended. But based on the planning discussions, many devs will be working on screen reader issues and they will likely need to bind new translatable texts that are generic enough to be used for aria-labels in any component. 
 Example: "Action cancelled, nothing changed"

So that's why I left it

Reply:
Okay. Better not to merge unused code or translations.